tests/api_tests/test_participant_summary_api.py

```diff
@@ -506,6 +506,16 @@ def test_invalid_filters_return(self):
         response_no_filter = self.send_get("ParticipantSummary")
         self.assertEqual(len(response_no_filter['entry']), num_summary)
 
+    def test_access_with_curation_role(self):
+        participant = self.send_post("Participant", {"providerLink": [self.provider_link]})
+        participant_id = participant["participantId"]
+        with FakeClock(TIME_1):
+            self.send_consent(participant_id)
```

Comment:
```suggestion
        self.data_generator.create_database_participant_summary()
```
nit: The `data_generator` can create a random participant summary more concisely to help a bit with readability.

Reply:
👍